### ISSUE TITLE
Umbrella header for module 'YapDatabase' does not include header ...

### ISSUE THREAD

**Issue by diehardest** [yapstudios/YapDatabase#208]
Hi Guys, and thank you for the great tool!

I have some strange warnings here, please see screenshot
![screen shot 2015-08-27 at 6 55 50 pm](https://cloud.githubusercontent.com/assets/9250530/9525540/4624d56c-4ced-11e5-8836-c7ed089e4a3b.png)
I'm using  YapDatabase installed via CocoaPods and including it in modern module way 
`@import YapDatabase;`

I would really appreciate it if you kindly give me some feedback with the issue

**Comment by just-a-beginner** [comment]
I have this same error - will comment if I find anything works to remove it.

**Comment by just-a-beginner** [comment]
Left an answer for you on SO. http://stackoverflow.com/questions/32268102/yapdatabase-warnings-umbrella-header-for-module-yapdatabase-does-not-include/32378266#32378266

pod update fixed it for me.

**Comment by diehardest** [comment]
Thank you, but that didn't help me.

**Comment by just-a-beginner** [comment]
Well, after saying it fixed it, now I am getting this error again. So, any help would assist me too!

**Comment by danthorpe** [comment]
I'm also seeing these warnings, using Cocoapods v0.38.2, Xcode 6.4, Swift 1.2 and YapDatabase 2.7.2.

**Comment by king7532** [comment]
Ditto @danthorpe, my Archive build using the "Release" configuration crashes the swift compiler in Xcode 6.4 (6E35b) with Swift 1.2, but "Debug" configuration does not crash:
```
<module-includes>:1:1: warning: umbrella header for module 'YapDatabase' does not include header 'YapDatabaseRTreeIndex.h'
#import "/Users/king/Library/Developer/Xcode/DerivedData/ABCWorkspace-eyiufvdusncvfwejohryurplvxbs/Build/Intermediates/ArchiveIntermediates/ABC/BuildProductsPath/Release-iphoneos/YapDatabase.framework/Headers/YapDatabase.h"
^
<module-includes>:1:1: warning: umbrella header for module 'YapDatabase' does not include header 'YapDatabaseRTreeIndexConnection.h'
#import "/Users/king/Library/Developer/Xcode/DerivedData/ABCWorkspace-eyiufvdusncvfwejohryurplvxbs/Build/Intermediates/ArchiveIntermediates/ABC/BuildProductsPath/Release-iphoneos/YapDatabase.framework/Headers/YapDatabase.h"
^
<module-includes>:1:1: warning: umbrella header for module 'YapDatabase' does not include header 'YapDatabaseRTreeIndexHandler.h'
#import "/Users/king/Library/Developer/Xcode/DerivedData/ABCWorkspace-eyiufvdusncvfwejohryurplvxbs/Build/Intermediates/ArchiveIntermediates/ABC/BuildProductsPath/Release-iphoneos/YapDatabase.framework/Headers/YapDatabase.h"
^
<module-includes>:1:1: warning: umbrella header for module 'YapDatabase' does not include header 'YapDatabaseRTreeIndexOptions.h'
#import "/Users/king/Library/Developer/Xcode/DerivedData/ABCWorkspace-eyiufvdusncvfwejohryurplvxbs/Build/Intermediates/ArchiveIntermediates/ABC/BuildProductsPath/Release-iphoneos/YapDatabase.framework/Headers/YapDatabase.h"
^
<module-includes>:1:1: warning: umbrella header for module 'YapDatabase' does not include header 'YapDatabaseRTreeIndexSetup.h'
#import "/Users/king/Library/Developer/Xcode/DerivedData/ABCWorkspace-eyiufvdusncvfwejohryurplvxbs/Build/Intermediates/ArchiveIntermediates/ABC/BuildProductsPath/Release-iphoneos/YapDatabase.framework/Headers/YapDatabase.h"
^
<module-includes>:1:1: warning: umbrella header for module 'YapDatabase' does not include header 'YapDatabaseRTreeIndexTransaction.h'
#import "/Users/king/Library/Developer/Xcode/DerivedData/ABCWorkspace-eyiufvdusncvfwejohryurplvxbs/Build/Intermediates/ArchiveIntermediates/ABC/BuildProductsPath/Release-iphoneos/YapDatabase.framework/Headers/YapDatabase.h"
^
<module-includes>:1:1: warning: umbrella header for module 'YapDatabase' does not include header 'YapDatabaseSearchQueuePrivate.h'
#import "/Users/king/Library/Developer/Xcode/DerivedData/ABCWorkspace-eyiufvdusncvfwejohryurplvxbs/Build/Intermediates/ArchiveIntermediates/ABC/BuildProductsPath/Release-iphoneos/YapDatabase.framework/Headers/YapDatabase.h"
^
0  swift                    0x000000010ea402b8 llvm::sys::PrintStackTrace(__sFILE*) + 40
1  swift                    0x000000010ea40794 SignalHandler(int) + 452
2  libsystem_platform.dylib 0x00007fff8f449f1a _sigtramp + 26
3  libsystem_platform.dylib 0x00007fff518d7770 _sigtramp + 3259553904
4  swift                    0x000000010e55270a (anonymous namespace)::SimplifyCFGPass::run() + 266
5  swift                    0x000000010e525f3e swift::SILPassManager::runFunctionPasses(llvm::ArrayRef<swift::SILFunctionTransform*>) + 1310
6  swift                    0x000000010e5269c9 swift::SILPassManager::runOneIteration() + 633
7  swift                    0x000000010e525436 swift::runSILOptimizationPasses(swift::SILModule&) + 790
8  swift                    0x000000010e3298e7 frontend_main(llvm::ArrayRef<char const*>, char const*, void*) + 4695
9  swift                    0x000000010e3284e6 main + 1814
10 libdyld.dylib            0x00007fff8edb35c9 start + 1
11 libdyld.dylib            0x0000000000000069 start + 1898236577
```

Cocoapods v0.38.2, Xcode 6.4, Swift 1.2 and YapDatabase 2.7.2

**Comment by danthorpe** [comment]
I've created a PR: #213 which adds the missing module headers.

**Comment by robbiehanson** [comment]
Merged PR #213
